saleor/settings.py

```diff
@@ -468,3 +473,15 @@ def get_host():
 DEFAULT_MENUS = {
     'top_menu_name': 'navbar',
     'bottom_menu_name': 'footer'}
+
+
+NOCAPTCHA = True
+
+ENABLE_RECAPTCHA = get_bool('ENABLE_RECAPTCHA', False)
```

Comment:
Wonder if we shouldn't enable it by default, @patrys what do you think?

Reply:
I'd say enable it if we have a key. In future we should likely move the key from the environment to the settings screen.